Update tutorial.md fix typo (#4735)
galaxyproject/training-material: topics/ecology/tutorials/Metashrimps_tutorial/tutorial.md

> <comment-title>What is a Data Paper?</comment-title>
> According to the [GBIF](https://www.gbif.org/data-papers) (Global Biodiversity Information Facility), 
> A data paper is a peer reviewed document describing a dataset, published in a peer reviewed journal. It takes effort to prepare, curate and describe data. 

> >,Data papers provide recognition for this effort by means of a scholarly article.

Reviewer: ```suggestion
> Data papers provide recognition for this effort using a scholarly article.
```

# Upload Data in MetaShRIMPS

> When opening [MetaShRIMPS](https://ecology.usegalaxy.eu/root?tool_id=interactive_tool_metashrimps), you will have an interface looking like this :

Reviewer 2: ```suggestion
When opening {% tool [MetaShRIMPS](interactive_tool_metashrimps) %}, you will have an interface looking like this :
```
